app/controllers/interviews_controller.rb

```diff
@@ -0,0 +1,72 @@
+class InterviewsController < ApplicationController
+  before_action :check_employee_role, only: [:new, :create, :edit, :update, :destroy!]
+  before_action :find_interview, only: %i[edit update destroy]
+
+  def index
+    interview_index
+  end
+
+  def new
+    @interview = current_employee.created_interviews.new
+  end
+  
+  def create
+    @interview = current_employee.created_interviews.new(interview_params)
+    if @interview.save
+      redirect_to interviews_path
+    else
+      render :new
+    end
+  end
+
+  def edit
+    @date =  @interview.scheduled_at.strftime("%Y-%m-%d")
```

Comment:
can we shift **interview.scheduled_at.strftime("%Y-%m-%d")** to view and use it directly?